com.unity.mobile.notifications/Tests/Runtime/Android/AndroidNotificationSendingTests.cs

```diff
@@ -292,4 +293,40 @@ public IEnumerator SendNotification_CanAccessNativeBuilder()
             Assert.AreEqual("TheTest", extras.Call<string>("getString", "notification.test.string"));
         }
     }
+
+    [UnityTest]
+    [UnityPlatform(RuntimePlatform.Android)]
+    public IEnumerator SendNotification_CanReschedule()
+    {
+        var managerClass = new AndroidJavaClass("com.unity.androidnotifications.UnityNotificationManager");
+        var rebootClass = new AndroidJavaClass("com.unity.androidnotifications.UnityNotificationRestartOnBootReceiver");
+        AndroidJavaObject context;
+        using (var unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer"))
+        {
+            using (var activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity"))
+                context = activity.Call<AndroidJavaObject>("getApplicationContext");
+        }
+
+        var n = new AndroidNotification();
+        n.Title = "SendNotification_CanReschedule";
+        n.Text = "SendNotification_CanReschedule Text";
+        n.FireTime = System.DateTime.Now.AddSeconds(5);
+
+        Debug.LogWarning("SendNotification_CanReschedule sends notification");
+
+        int id = AndroidNotificationCenter.SendNotification(n, kDefaultTestChannel);
+        yield return new WaitForSeconds(0.2f);
```

Comment:
Probably nothing, but lately all tests that schedule were plagued with instabilities, so I didn't want to add yet one more. This wait brings the a test a little bit closer to reality, since sending and canceling immediately is very far from device reboot while notification is scheduled.